src/components/BookItem.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable camelcase */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] [REQUIRED] I can see that you are also disabling the Javascript camelCase convention here. Kindly enable Eslint because ESLint enforces a consistent coding style across the project, ensuring that all code follows the same standards. Let's fix this as well.

Reply:
Fixed.

Reply:
- Well resolved 👍🏽